docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -372,4 +372,13 @@ Semgrep provides the following environments to learn, experiment, and write Semg
 
 Refer to [Troubleshooting Semgrep App](../troubleshooting/semgrep-app/) for common installation issues. Help is also available for all users through the [r2c Community Slack](https://r2c.dev/slack).
 
+## Additional resources
+
+### Semgrep App session details
+
+- The time before you need to reauthenticate to Semgrep App is 7 days.
+- Semgrep App session token is valid for 7 days.
+- This session timeout is not configurable.
+- Semgrep App is not using cookies but `localStorage` to store access tokens. The data in `localStorage` expire every 7 days. 

```

Comment:
@milanwilliams I am using `localStorage` marked as inline code similar to https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage